python/console/console_editor.py

```diff
@@ -229,6 +272,93 @@ def findNext(self):
     def findPrevious(self):
         self.findText(False)
 
+    def textBeforeCursor(self):
+        return self.text(0, self.positionFromLineIndex(*self.getCursorPosition()))
+
+    def reformatCode(self):
+        """ Reformat the code using the selected formatter """
+
+        formatter = self.settings.value("pythonConsole/formatter", "autopep8", type=str)
+        max_line_length = self.settings.value("pythonConsole/maxLineLength", 80, type=int)
```

Comment:
Agreed that the 80 characters max line length is a bit conservative, but why use the black default value? The 80 limt was already set in 
https://github.com/qgis/QGIS/blob/19d7edcfedd492b3489260aaf8febe78fcb882e5/src/gui/codeeditors/qgscodeeditorpython.cpp#L67-L74

This can always be overriden in the settings, so I think the default value should stay as it is.